src/telemetry/metrics.yaml

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
```

Comment:
As someone unfamiliar with this configuration file: could we also add a link to the docs on available options here?